mycroft/skills/phillips_hue/vocab/en-us/ConnectLIghtsKeyword.voc

```diff
@@ -0,0 +1,8 @@
+connect my lights
+connect the light
+connect to my lights
+connect to the lights
+reconnect my lights
+reconnect the light
+reconnect to my lights
+reconnect to the lights
```

Comment:
To clean up this vocab, you can register a `LightKeyword.voc` that has lights and light in it, and then change this to be a `ConnectKeyword.voc` that just has connect and reconnect in it. Take a look at the [Volume skill](https://github.com/MycroftAI/mycroft-core/tree/master/mycroft/skills/volume) for a good example of how to do this.